conf/modules/imu_cube.xml

```diff
@@ -10,7 +10,7 @@
     </description>
   </doc>
   <dep>
-    <depends>spi_master,imu_common</depends>
+    <depends>spi_master,imu_common,intermcu_iomcu,imu_heater</depends>
```

Comment:
Yes I'm currently not flashing the IOMCU and only flashing the FMU. Mainly because I haven't spend any time making a board file for it and also the flashing has to happen through the FMU.
For now indeed this means you can't access the servo and RC pins which are connected to the IOMCU. The FMU also has PWM outputs and has more than enough UARTs to connect an RC. Next to that we mainly use CAN anyways.
Yes this would block someone now for using the IO mcu, but I think when someone wants to use it properly he can remove the full `intermcu_iomcu` and we just implement the heater through our own FBW. This was just easier and faster, since I only wanted the heater option.